srml/contracts/src/exec.rs

```diff
@@ -120,6 +120,9 @@ pub trait Ext {
 
 	/// Rent allowance of the contract
 	fn rent_allowance(&self) -> BalanceOf<Self::T>;
+
+	// Returns a reference to the current block number.
+	fn current_block(&self) -> &<Self::T as system::Trait>::BlockNumber;
```

Comment:
nitpick but why not call this `block_number`?

Reply:
Does this question mean about naming of function?

Reply:
Yep, I'm suggesting replacing this with `fn block_number(&self) -> BlockNumberOf<Self::T>;`